src/main/java/org/myrobotlab/service/config/LogConfig.java

```diff
@@ -0,0 +1,12 @@
+package org.myrobotlab.service.config;
+
+public class LogConfig extends ServiceConfig {
+
+  public String currentUserName;
```

Comment:
why does log config care about currentUserName?

Reply:
copy paste config from programab .. trying to keep up get services their own real config vs ServiceConfig 
removing...